HelpSource/Classes/Object.schelp

```diff
@@ -692,8 +692,129 @@ code::
 method::dump
 
 Print a detailed low level representation of the receiver to the post window.
+Except for the link::Classes/List:: class, this method is not overridden in other classes. Any of the MetaClasses, Classes and Instances can be used by this method.
+
+code::
+Meta_Object.dump // the meta class of the class Object
+::
+code::
+Object.dump      // the class called Object
+::
+code::
+Object.new.dump  // an istance of the class Object
+::
+
+discussion::
+
+list::
+## The detailed low level format and information varies depending on the receiver.
+## Some instance objects, especially unique objects, return the class name and value (also low data if necessary) of the dumped object:
+
+definitionlist::
+## link::Classes/Float::
+||
+code::
+1.0.dump
+::
+64-bit version of SuperCollider returns:
+teletype::
+   Float 1.000000   00000000 3FF00000
+-> 1.0
+::
+The last two groups of an 8-digit integer are the raw hexadecimal representation of the 64-bit double value according to link::https://ieeexplore.ieee.org/document/8766229##IEEE 754 Floating Point:: (https://ieeexplore.ieee.org/document/8766229). Each part is represented as follows:
+
+teletype::
+                         raw hexadecimal representation
+                           of the 64-bit double value
+                         ––––––––––––––––––––––––––––––
+Float   -1.000000        00000000              3FF00000
+|       |                |                     |
+class   decmial          significant part      exponent part
+        representation   (mantissa)            with sign bit
+::
+## link::Classes/Integer::
+||
+code::
+1.dump
+::
+teletype::
+   Integer 1
+-> 1
+::
+## link::Classes/Char::
+	||
+code::
+$1.dump
+::
+teletype::
+   Character 49 '1'
+-> 1
+::
+The integer between emphasis::Character:: and emphasis::'1':: is the ASCII value of that character.
+## link::Classes/Symbol::
+||
+code::
+\1.dump
+::
+teletype::
+   Symbol '1'
+-> 1
+::
+::
+
+## Some instance objects return more detailed information, such as
+list::
+## addrress in virtual memory (the hexadecimal number prefixed with 0x),
+## emphasis::garage collector color:: (gc),
+## emphasis::data format type:: (fmt),
+## emphasis::flags for immutablity, finalization and garbage collector debug sanity check:: (flg),
```

Comment:
sanity information from the garbage collector

Reply:
Thank you, done! But could you check it again, if I understand you correctly?